docs/other_vscode.md

```diff
@@ -0,0 +1,117 @@
+# Setting up Visual Studio Code for QMK Development
+
+Visual Studio Code ([VS Code](https://code.visualstudio.com/) is an open-source [Integrated Development Environment](https://en.wikipedia.org/wiki/Integrated_development_environment) (IDE), that supports many different programming languages. 
```

Comment:
Although VS Code blurs the line between editor and IDE a bit, it's not an IDE, since it mainly accomplishes this through extensions. Microsoft doesn't call it an IDE either; they insist that it's a code editor.

```suggestion
[Visual Studio Code](https://code.visualstudio.com/) (VS Code) is an open-source code editor that supports many different programming languages. 
```

Also, I think the link should go on the full name.